src/pages/Signup.tsx

```diff
@@ -77,6 +179,9 @@ const Signup = (): JSX.Element => {
           <br />
           <button>Sign Up</button>
         </form>
+        {show && (
```

Comment:
Seeing the use case for the `show` useState, could have joined both into one (**suggestion**).

Maybe could have used the `"status"` of the `formState`, something like: 
```ts
{formState.status !== 'pending` && ...}
```
Or leave it empty so can be used as a falsy value

```ts
const [formState, setFormState] = {status: '', ...} //assuming is an empthy string...
{!formState.status && ...}